packages/mobx-react/src/observerClass.ts

```diff
@@ -90,12 +90,12 @@ export function makeClassComponentObserver(
             // Forces reaction to be re-created on next render
             this.render[mobxAdminProperty] = null
         } else {
-            // Render may have been hot-swapped and/or overriden by a subclass.
+            // Render may have been hot-swapped and/or overridden by a subclass.
             const displayName = getDisplayName(this)
             console.warn(
                 `The reactive render of an observer class component (${displayName})
-                was overriden after MobX attached. This may result in a memory leak if the
-                overriden reactive render was not properly disposed.`
+                was overridden after MobX attached. This may result in a memory leak if the
```

Comment:
Please add a changeset `yarn changeset` mentioning these improvements so it forces release and you get a mention in release notes, which is always nice :D

Reply:
Isn't this too minor to publish as a release? Would be perfectly okay with me that it just come with the next release, however I can do that if that is the norm.

Reply:
Your change will be just a part of the release. So feel free to add it 👍